probes/releasesHaveVerifiedProvenance/def.yml

```diff
@@ -0,0 +1,34 @@
+# Copyright 2024 OpenSSF Scorecard Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+id: releasesHaveVerifiedProvenance
+short: Checks if the project releases with provenance attestations
```

Comment:
compared to the existing `releasesHaveProvenance`, this might create some confusion.

Reply:
It may be worth mentioning the word "package" somewhere, since this is currently for package manager releases, not a GitHub/GitLab release.

Does `packagesPublishedWithProvenance` or something make sense to distinguish the two types of releases?

Reply:
Yes, but the implementation of this might evolve to include GitHub releases, since there are SLSA provenance actions and also GitHub actions' new attestation publishing feature. This probe would capture all of those. We don't have a standard place to look for those but once we do they could be part of this probe!

Reply:
We'd ideally have a single "view" of releases b/w releases on the SCM and the package manager, so not creating distinctions b/w this and the other probes is somewhat intentional.